src/libs/security-hub-lib.ts

```diff
@@ -52,6 +57,12 @@ export class SecurityHub {
         ],
         ProductName: [{ Comparison: "EQUALS", Value: "Security Hub" }],
         SeverityLabel: this.severityLabels,
+        CreatedAt: [
```

Comment:
MC-Review decided to point this tool at their prod environment, so ignoring findings that are less than one day old doesn't make sense for them since it just knocks a day off the time they have left to remediate the finding.  What do you think about making this an option that is passed in to the package and set as a property on the `SecurityHub` class?  It could be called something like `ignoreEphemeralFindings` and default to true, then MC-Review could override it. Or it could be opt-in, whatever makes more sense to you guys